NEWS: add a belated NEWS file with a few entries for 0.9.1

trofi opened the pull request:
The main focus of this release is on medialib migration.

nesciens commented on NEWS:
```diff
@@ -0,0 +1,32 @@
+
+* Medialib database switched from sqlite3 backend to s4 library backend.
+
+  For typical xmms2 setups that run in home directories backend
```
```suggestion
  For typical xmms2 setups that run in home directories, backend
```

nesciens reviewed (approved):
Thanks for this, and for testing the migration!

Mostly non-blocking textual comments from me, and a question about sqlite2s4.

Please reject the textual tweaks if you feel they erase your tone of voice.

---

This is my first review via GitHub, so I might be holding this wrong. Tips for doing this more smoothly welcome!

trofi commented on NEWS:
```diff
@@ -0,0 +1,32 @@
+  For typical xmms2 setups that run in home directories backend
+  migration will happen automatically: once new version of `xmms2d`
+  starts it will convert ~/.config/xmms2/medialib.db sqlite database
+  to ~/.config/xmms2/medialib.s4 s4 database by running `sqlite2s4`
```
Good idea! Added a standing out bullet point right after migration section as:

```
* New `sqlite2s4` tool was added. `xmms2d` expects it's presence in
  `PATH` to perform automatic medialib conversion to s4 format.
```

trofi commented:
> Mostly non-blocking textual comments from me, and a question about sqlite2s4.

Pulled all you suggestions in, added a few newlines and squashed into a single commit. Please take another look.

Thank you!

nesciens commented on NEWS:
```diff
@@ -0,0 +1,39 @@
+  Once the conversion is done, the sqlite database will move to
+  ~/.config/xmms2/medialib.db.obsolete.
+
+* New `sqlite2s4` tool was added. `xmms2d` expects it's presence in
+  `PATH` to perform automatic medialib conversion to s4 format.
```
```suggestion
* The new `sqlite2s4` tool was added. `xmms2d` expects its presence in
  `PATH` to perform automatic medialib conversion to the s4 format.
```

nesciens reviewed (approved):
A suggestion for the commit message (textual and collection vs medialib):

```
- Main focus of ths release is on collection migration.
+ The main focus of this release is on medialib migration.
```

Otherwise looks good to me.

trofi commented:
> A suggestion for the commit message (textual and collection vs medialib):
> 
> ```
> - Main focus of ths release is on collection migration.
> + The main focus of this release is on medialib migration.
> ```

Good catch! Updated commit message and PR description.

Thank you!

nesciens reviewed (approved):
LGTM! 👍
